vision/tests/unittests/test_image_classification.py

```diff
@@ -64,3 +64,11 @@ def test_invalid_image_dataset():
     df = ImagePredictor.Dataset.from_csv(os.path.join(invalid_test, 'train.csv'), root=os.path.join(invalid_test, 'train_images'))
     predictor = ImagePredictor(label="labels")
     predictor.fit(df, df.copy(), time_limit=60)
+
+
+def test_image_predictor_presets():
+    ImagePredictor = Task
+    train_dataset, _, test_dataset = ImagePredictor.Dataset.from_folders('https://autogluon.s3.amazonaws.com/datasets/shopee-iet.zip')
+    for preset in ['best_quality', 'good_quality_fast_inference', 'medium_quality_faster_train', 'medium_quality_faster_inference']:
```

Comment:
It's probably only practical to do medium quality. best_quality would take many hours as currently implemented. time_limit also has to be specified and be a low value (such as 60/120s)